src/clients/SnapshotApi.ts

```diff
@@ -150,28 +150,30 @@ export class SnapshotApi {
     return (await this.client.vote(account, address, voteMessage)) as SnapshotReceipt
   }
 
-  async getScores(addresses: string[], blockNumber?: number | string, space?: string, networkId?: string) {
+  async getScores(
+    addresses: string[],
+    blockNumber?: number | string,
+    space?: string,
+    networkId?: string,
+    proposalStrategies?: SnapshotStrategy[]
+  ) {
     const formattedAddresses = addresses.map((address) => getChecksumAddress(address))
     const network = networkId && networkId.length > 0 ? networkId : getEnvironmentChainId().toString()
     const spaceName = space && space.length > 0 ? space : SnapshotApi.getSpaceName()
     const snapshotSpace = await SnapshotGraphql.get().getSpace(spaceName)
+    const strategies = proposalStrategies ? proposalStrategies : snapshotSpace.strategies
```

Comment:
Are we using snapshotSpace for something else? Maybe you can do something like....
`const strategies = proposalStrategies || (await SnapshotGraphql.get().getSpace(spaceName)).strategies`